core/build.gradle.kts

```diff
@@ -205,6 +205,7 @@ val processKDocsMain by creatingProcessDocTask(
         ARG_DOC_PROCESSOR,
         COMMENT_DOC_PROCESSOR,
         SAMPLE_DOC_PROCESSOR,
+//        REMOVE_ESCAPE_CHARS_PROCESSOR,
```

Comment:
?

Reply:
will be in the next release of the doc preprocessor, I must not forget to enable it then :)